cluster/gce/config-default.sh

```diff
@@ -95,6 +95,12 @@ MASTER_EXTRA_METADATA=${KUBE_MASTER_EXTRA_METADATA:-${KUBE_EXTRA_METADATA:-}}
 NODE_EXTRA_METADATA=${KUBE_NODE_EXTRA_METADATA:-${KUBE_EXTRA_METADATA:-}}
 # KUBELET_TEST_ARGS are extra arguments passed to kubelet.
 KUBELET_TEST_ARGS=${KUBE_KUBELET_EXTRA_ARGS:-}
+if [[ "${CONTAINER_RUNTIME}" == "containerd" ]]; then
```

Comment:
I'm torn between leaving this as it is, or grouping the block with the docker settings above.

Reply:
Like this?